standards/storing-source-code.md

```diff
@@ -30,6 +30,30 @@ Ask your delivery manager to request you being added to the Github organisation.
 Repositories should be [clearly named]({{ '/standards/naming-things' | relative_url }}),
 and have an [appropriate licence]({{ '/standards/licencing-software-or-code' | relative_url }})
 and enough documentation that someone new can get started with the
-project.
+project. They should be initialised using the [das-github-template]{:target="_blank"} (WIP) (see: [creating a repository from a template]{:target="_blank"} and contain code that will be built and deployed through a single build and release pipeline. 
+
+At a minimum the repository should:
+
+- not be a fork
+- use [branch protection rules]{:target="_blank"} that
+    - require pull requests reviews
+    - require status checks on commits
+- receive [dependabot alerts and security updates]{:target="_blank"}
+- assign the following teams accessassign the das-contributor team as write
+- assign the DevOps team as admin
+- assign the Security team as reader
+- include a webhook to receive notifications of repository vulnerability alerts
+- make use of:
+    - the [CODEOWNERS]{:target="_blank"} file which should be updated regularly as the team responsible for maintaining the repository changes
```

Comment:
A CODEOWNERS file isn't used across all apps' repositories. Some app teams not require it.

Reply:
That's true, but is it something we want to start using as a standard going forward?

Reply:
I think maybe best left to the app teams to decide if they want one.

Reply:
Removed